src/main/java/com/meilisearch/sdk/TypoTolerance.java

```diff
@@ -0,0 +1,60 @@
+package com.meilisearch.sdk;
+
+import java.util.HashMap;
+import org.json.JSONObject;
+
+public class TypoTolerance {
+
+    public TypoTolerance() {}
+
+    protected boolean enabled = false;
+    protected HashMap<String, Integer> minWordSizeForTypos;
+    protected String[] disableOnWords;
+    protected String[] disableOnAttributes;
```

Comment:
In the class `public class Settings {` you used `@Getter` and `@Setter`, this can be applied here as well, right?

Reply:
Yes, it depends on the class some have `getter` and `setter wrote as a function, and other has `@Getter @Setter`.I'm not sure to understand exactly why but I try to keep the same as the other class file like `Key` or `Details` class. Maybe we should create an issue to harmonize everything?

Reply:
I think you should definitely create the issue, but also do the changes (for the files you touched in this branch) now.

I was looking for some references to this particular topic: "wait to do all the refactor at once or do incrementally", and I found this https://meiert.com/en/blog/boyscout-code/, a good reading!

Reply:
I open an issue about it #379